code/modules/hydroponics/seeds.dm

```diff
@@ -59,6 +59,8 @@
 	var/grafted = FALSE
 	///Type-path of trait to be applied when grafting a plant.
 	var/graft_gene
+	///Determines if the plant should be allowed to mutate early at 30+ instability.
+	var/seed_flags = TRUE
```

Comment:
```suggestion
	///Bitflags for boolean seed values, such as MUTATE_EARLY
	var/seed_flags = NONE
```
I assume you don't want things to mutate by default.

Reply:
No, that's the goal here, the issue is that there are specific plants whose product when mutated (Clonepods from cabbage) that will run into issues when mutated, which really made it more akin to an exception. I can go and swap the bigflag over to something like `NO_MUTATION_EARLY` instead in that case